CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Changes from version 0.10.3 to 0.10.4
+# Changes from version 0.10.3 to master
+
+- Added support for log file rotation for started server components.
+
+# Changes from version 0.10.4 to 0.10.4
```

Comment:
Looks like it should be "0.10.3 to 0.10.4"

Reply:
both done